x/committee/types/msg_test.go

```diff
@@ -60,8 +60,8 @@ func TestMsgVote_ValidateBasic(t *testing.T) {
 			expectPass: true,
 		},
 		{
-			name:       "abstain",
-			msg:        MsgVote{5, addr, Abstain},
+			name:       "No",
+			msg:        MsgVote{5, addr, No},
 			expectPass: true,
 		},
```

Comment:
invalid vote test needs to be updated

Reply:
https://github.com/Kava-Labs/kava/pull/902/commits/37ef810df270191d0fa473ad25c5a76eb568273e